src/primitives/transaction.cpp

```diff
@@ -183,15 +173,15 @@ bool CTransaction::HasP2CSOutputs() const
 CAmount CTransaction::GetValueOut() const
 {
     CAmount nValueOut = 0;
-    for (std::vector<CTxOut>::const_iterator it(vout.begin()); it != vout.end(); ++it) {
+    for (const CTxOut& it : vout) {
```

Comment:
μ-nit: better to call it `out` rather than `it` as it's no longer an iterator.

Reply:
sure, one sec